cmd/root.go

```diff
@@ -93,6 +93,9 @@ func loadConfiguration() *config.Config {
 	k9sCfg.K9s.OverrideReadOnly(*k9sFlags.ReadOnly)
 	k9sCfg.K9s.OverrideWrite(*k9sFlags.Write)
 	k9sCfg.K9s.OverrideCommand(*k9sFlags.Command)
+	k9sCfg.K9s.OverrideDumpDirPath(*k9sFlags.DumpDirPath)
+
+	k9sCfg.InstantiateK9sDumpDir()
```

Comment:
Don't think this should be exposed. Best to deal with that once the args and overrides are actually resolved.

Reply:
Removed from the function `loadConfiguration`